org.telegram.desktop.json

```diff
@@ -0,0 +1,288 @@
+{
+    "id": "org.telegram.desktop",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "telegram-desktop",
+    "rename-desktop-file": "telegramdesktop.desktop",
+    "rename-appdata-file": "telegramdesktop.appdata.xml",
+    "rename-icon": "telegram-desktop",
+    "finish-args": [ "--share=ipc",
+                     "--share=network",
+                     "--socket=x11",
+                     "--socket=pulseaudio",
+                     "--filesystem=home",
```

Comment:
Does a chat app really need full home access? (I assume that Qt supports the portals for file selection too)